### ISSUE TITLE
isPrecompiled() needs to be updated to reflect byzantium

### ISSUE THREAD

**Issue by axic** [ethereumjs/ethereumjs-util#110]
See https://github.com/ethereumjs/ethereumjs-util/blob/master/index.js#L519, which limits it to address 0 to 5.

(This feature is used by the -vm and -accounts.)

**Comment by axic** [comment]
Update: 
- apparently it is not used by `accounts` anymore for a while: https://github.com/ethereumjs/ethereumjs-account/pull/2
- the VM does its check internally

This feature could be removed in the next breaking change, but for now it still needs to be updated.

**Comment by holgerd77** [comment]
Yes, removed this from the VM when working on Byzantium. Found this too implicit with too few benefits.
